docs/content/latest/benchmark/tpcc-ysql/100-wh/tpcc-ysql.md

```diff
@@ -1,9 +1,6 @@
-### TPC-C Load Phase
+## TPC-C Load Phase
 
-Before starting the workload, you will need to load the data first. Make sure
-to replace the IP addresses with that of the nodes in the cluster. Loader
-threads allow us to configure the number of threads used to load the data. For
-a 3 node c5d.4xlarge cluster, loader threads value of 48 was the most optimal.
+Before starting the workload, you need to load the data. Make sure to replace the IP addresses with that of the nodes in the cluster. Loader threads allow you to configure the number of threads used to load the data. For a 3-node c5d.4xlarge cluster, loader threads value of 48 was the most optimal.
 
```

Comment:
a loader threads value of 48 was optimal.

Reply:
Fixed